airbyte-integrations/connectors/source-facebook-marketing/setup.py

```diff
@@ -6,9 +6,9 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1.35",
-    "cached_property~=1.5",
-    "facebook_business~=12.0",
+    "airbyte-cdk==0.1.47",
```

Comment:
why not `~=`? doesn't that auto-bump the dep versions?

Reply:
it is not, and it shouldn't see https://github.com/airbytehq/airbyte/issues/9697